releng_changelog.md

```diff
@@ -6,6 +6,8 @@ This page best serves the people that have previously been into the releaseduty
 
 _**As releaseduty squirrels are the ones with the best context when it comes to releases, they are the ones to edit this page and amend it accordingly. Keep in mind that changes should be in compliance with the other pieces of documentation.**_
 
+### During 107.0 >= 2022-08-20
```

Comment:
Typo: s/entrey/entry/ in the commit message.

Reply:
Indeed! 👍